### DISCUSSION TITLE
is there any way to access filesystem easily without cd command every time?

### DISCUSSION THREAD

**Discussion by imtheaman** [helix-editor/helix#2538] [Q&A] [answered]
i have to access files/change files by going back every time and using cd command which for continuous development is time counter-productive. is there any simple way to access the entire folder as we were using in vs code. files and folders in a tree format inside the editor...

initially i get it but when i open a file it disappear
![Screenshot_connect-server : hx_1](https://user-images.githubusercontent.com/62852336/169741065-510fee29-296c-4d0a-b086-1629c66ddbf0.png)

**Comment by archseer** [comment, marked as answer]
`Space + f`

**Reply by imtheaman** [reply]
Thank you.
